extensions/amp-analytics/amp-analytics.md

```diff
@@ -59,6 +59,13 @@ when the document is first loaded, and each time an `<a>` tag is clicked:
 </amp-analytics>
 ```
 
+#### <a name="attributes"></a>Attributes
+
+  - `type` This optional attribute can be specified to use one of the built-in analytics providers. Currently supported types:
+    - `googleanalytics`: This type defines the basic requests like `pageview`,  `event`, `social` and `timing`.
```

Comment:
@rudygalfi Removed the line. Added a link to details docs. (doc is still being written).

@Meggin: Does this look better?